src/mono/cmake/config.h.in

```diff
@@ -8,7 +8,6 @@
 #pragma warning(disable:4090) // const problem
 #pragma warning(disable:4146) // unary minus operator applied to unsigned type, result still unsigned
 #pragma warning(disable:4244) // integer conversion, possible loss of data
-#pragma warning(disable:4267) // integer conversion, possible loss of data
```

Comment:
I wonder how painful it would be to enable `-Wshorten-64-to-32` on clang - it would help catch some of these errors when building on OSX.  Not a lot of the mono team works with the Win32 builds

Reply:
1674 hits on M1 (osx-arm64).. project for another weekend. 😅

Reply:
On main branch 2002 hits.